content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -184,12 +184,9 @@ export class _CollapsibleSection extends React.PureComponent {
             <button
               className="context-menu-button icon"
               title={this.props.intl.formatMessage({id: "context_menu_title"})}
+              aria-label={this.props.intl.formatMessage({id: "context_menu_title"})}
               onClick={this.onMenuButtonClick}
-              ref={this.setContextMenuButtonRef}>
-              <span className="sr-only">
-                <FormattedMessage id="section_context_menu_button_sr" />
```

Comment:
Is this an intended change from `section_context_menu_button_sr` to `context_menu_title` ?

Reply:
I had wanted the titles and aria-labels to have consistent content, but `section_context_menu_button_sr` is actually probably a better choice for both of them, since the message is more detailed. I'll switch it over in all these cases.

Reply:
So `section_context_menu_button_sr` doesn't seem to pull the {title} prop through correctly - I get a lot of "open context menu for {title}." , even when I give the component a hardcoded prop. Is there somewhere else it needs to be set as well?

Reply:
Getting @wolasi feedback as this is a user facing change and could be tricky with localized long text as below:

<img width="1256" alt="tooltip" src="https://user-images.githubusercontent.com/597734/59302782-8dbd3780-8c49-11e9-8c37-9d2ffadee04a.png">

Reply:
I think the previous default of "Open menu" is fine for the tooltip

Reply:
okay, I'll make that change :)